tests/unit/services/cookies-test.js

```diff
@@ -74,6 +74,11 @@ describe('CookiesService', function() {
         expect(afterRoundtrip).to.eq(value);
       });
 
+      it('handles invalid values for cookies', function() {
+        document.cookie = '=blank';
+        expect(this.subject().read('blank')).to.be.blank;
```

Comment:
This should be `expect(this.subject().read('')).to.be.blank;` as the name of the cookie would be `''` without this fix.